src/Collectors/HttpRequestCollector.php

```diff
@@ -79,4 +73,9 @@ protected function getController(): ?string
 
         return $controller;
     }
+
+    private function isHttpRequest(): bool
+    {
+        return false !== strpos($_SERVER['SCRIPT_NAME'], 'index.php');
```

Comment:
Wouldn't [`php_sapi_name`](https://www.php.net/manual/en/function.php-sapi-name.php) or [`PHP_SAPI`](https://www.php.net/manual/en/reserved.constants.php#reserved.constants.core) do the same thing?

Reply:
I suppose so. Based on https://github.com/philkra/elastic-apm-laravel/issues/19, I also went ahead and added the option to disable non-HTTP while leaving HTTP logging active.